pkg/plugin/v3/scaffolds/internal/templates/config/crd/enablewebhook_patch.go

```diff
@@ -37,12 +40,35 @@ func (f *EnableWebhookPatch) SetTemplateDefaults() error {
 	}
 	f.Path = f.Resource.Replacer().Replace(f.Path)
 
-	f.TemplateBody = enableWebhookPatchTemplate
+	switch f.CRDVersion {
+	case v1beta1:
+		f.TemplateBody = enableWebhookPatchTemplatev1beta1
+	default:
+		f.TemplateBody = enableWebhookPatchTemplatev1
+	}
 
 	return nil
 }
 
-const enableWebhookPatchTemplate = `# The following patch enables conversion webhook for CRD
```

Comment:
I think we should not remove the comment

Reply:
v1 CRDs were introduced in k8s v1.16, so a comment about requiring v1.13+ in a v1 CRD manifest is irrelevant.

Reply:
i understand that the comment explains what it does. Sorry, I could not check the relation with the api version and the comment.